lib/jxl/enc_ac_strategy.cc

```diff
@@ -453,13 +453,17 @@ Status EstimateEntropy(const AcStrategy& acs, float entropy_mul, size_t x,
                                       (iy * kBlockDim + dy) *
                                           (acs.covered_blocks_x() * kBlockDim) +
                                       ix * kBlockDim + dx);
-              auto masku = Abs(Load(
-                  df8, config.MaskingPtr1x1(x + ix * 8 + dx, y + iy * 8 + dy)));
-              in = Mul(masku, in);
-              in = Mul(in, in);
-              in = Mul(in, in);
-              in = Mul(in, in);
-              lossc = Add(lossc, in);
+              if (x + ix * 8 + dx + Lanes(df8) <
```

Comment:
Shouldn't that be <=, and compare with the actual x size of the image?

This way, adding more padding will introduce the problem again.

Reply:
done